frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionDetailsPage.tsx

```diff
@@ -15,18 +15,6 @@ const menuActions = [
 export const NetworkAttachmentDefinitionsDetailsPage: React.FC<
   NetworkAttachmentDefinitionsDetailPageProps
 > = (props) => {
-  const { name, namespace } = props;
-
-  const resources = [
```

Comment:
This isn't needed. The resource will already be watched for details pages. This will watch it twice.